prow/nginx-ingress/ci/daemonset-tcp-udp-configMapNamespace-values.yaml

```diff
@@ -3,12 +3,12 @@ controller:
   service:
```

Comment:
these changes are not required. this directory is just a copy of the nginx chart, and this configmap in particular is never used by us

Reply:
Fixed 52b4eca